core/zoe/contracts/automaticRefund.js

```diff
@@ -11,18 +11,26 @@ import harden from '@agoric/harden';
  * @param {contractFacet} zoe - the contract facet of zoe
  */
 export const makeContract = harden((zoe, terms) => {
-  let count = 0;
-  const automaticRefund = harden({
-    makeOffer: async escrowReceipt => {
-      const { offerHandle } = await zoe.burnEscrowReceipt(escrowReceipt);
-      count += 1;
-      zoe.complete(harden([offerHandle]));
-      return `The offer was accepted`;
-    },
-    getOffersCount: () => count,
-  });
+  const initialInviteHandle = harden({});
+  let offersCount = 0;
+
+  const makeSeat = inviteHandle =>
+    harden({
+      makeOffer: () => {
+        offersCount += 1;
+        zoe.complete(harden([inviteHandle]), terms.assays);
+        return `The offer was accepted`;
+      },
+      makeInvite: () => {
+        const newInviteHandle = harden({});
+        const seat = makeSeat(newInviteHandle);
+        return zoe.makeInvite(seat, newInviteHandle);
+      },
+      getOffersCount: () => offersCount,
+    });
   return harden({
-    instance: automaticRefund,
+    initialSeat: makeSeat(initialInviteHandle),
+    initialInviteHandle,
```

Comment:
I'm surprised that you don't return the initial invite. AFAICT, there is no initial invite, in which case it is weird that there is an `initialInviteHandle`.

Reply:
there is an initial invite, but it's made by Zoe as part of the contract instantiation step, based on this  `initialInviteHandle` and the `initialSeat`. 

Let me see what I can do about making this simpler